docs/source/miniconda.rst.jinja2

```diff
@@ -61,10 +106,8 @@ Installing
 Other resources
 ===============
 
- -  `Miniconda with Python 3.8 for Power8 &
+ -  `Miniconda with Python 3.9 for Power8 &
```

Comment:
The Power Miniconda installer links are in the table above now, I don't think we need to call them out again in the Other resources section any more.

Reply:
This should be removed now. 👍